scripts/ci.sh

```diff
@@ -57,17 +57,19 @@ until curl -s -o/dev/null http://localhost:"$CI_WEB_PORT"; do
   fi
 done
 
-yarn db-seed-fresh
+# yarn db-seed-fresh
 
-if [ -z "$CYPRESS_ONLY" ]; then
-  craco test --runInBand --coverage
+# if [ -z "$CYPRESS_ONLY" ]; then
+  # craco test --runInBand --coverage
 
   # these are redundant with jest & cypress tests now
   # yarn --cwd hardhat test
-fi
+# fi
 
-if [ -z "$LOCAL" ]; then
-  yarn cypress run ${OTHERARGS[@]}
-else
-  yarn cypress open > /dev/null
-fi
+# if [ -z "$LOCAL" ]; then
+#   yarn cypress run ${OTHERARGS[@]}
+# else
+#   yarn cypress open > /dev/null
+# fi
+
+yarn jest api-test/__tests__
```

Comment:
I think these changes should be undone

Reply:
@karelianpie I can take care of this once I'm back. My plan is to merge this into the other branch anyway